### DISCUSSION TITLE
Partners PAGE

### DISCUSSION THREAD

**Discussion by dbwebapp** [BitCannaGlobal/WeedWallet#77] [Ideas]
A dedicated page for partners could be an opportunity to show up some good deal with your partners :

![Screenshot 2023-05-24 at 21 46 22](https://github.com/BitCannaGlobal/WeedWallet/assets/96944913/f2a4024d-efc1-4e13-8f50-9e2e35d5669f)

As an example, we can consider implementing a feature that detects if a connected user has any unclaimed rewards. If rewards are available, we can calculate the corresponding amount and suggest potential purchases with those rewards. For instance, a message could be displayed saying, "Hey, you have received this amount of BCNA as rewards. You can use it to get products like CBD oil or any other eligible items." Users tend to be more comfortable spending tokens they received as rewards, as they haven't directly paid for them. This approach can potentially attract a percentage of new buyers from the WeedWallet business, thereby generating more business opportunities and a % could be generated from the sell from WeedWallet.

**Comment by TJ-BitCanna** [comment]
Good idea!

This will not be in the first release, but we'll note it down for a future release.
